src/views/workflow-history/workflow-history-event-status-badge/workflow-history-event-status-badge.types.ts

```diff
@@ -0,0 +1,8 @@
+import { type WORKFLOW_EVENT_STATUS } from './workflow-history-event-status-badge.constants';
+
+export type WorkflowEventStatusBadgeSize = 'small' | 'medium';
+
+export type Props = {
+  status: (typeof WORKFLOW_EVENT_STATUS)[keyof typeof WORKFLOW_EVENT_STATUS];
```

Comment:
Is there no Cadence IDL type we can leverage for this?

Reply:
If not, I would rather have a union type here and build the WORKFLOW_EVENT_STATUS object using that type, instead of parsing the type from the constant